lib/redirect_by_addr_family.php

```diff
@@ -1,12 +1,20 @@
 <?php
-$loc_ipv6 = filter_input(INPUT_GET, 'ipv6', FILTER_SANITIZE_URL);
 $loc_ipv4 = filter_input(INPUT_GET, 'ipv4', FILTER_SANITIZE_URL);
-if(strpos($loc_ipv4, ':') || strpos($loc_ipv6, ':')) {
+$loc_ipv6 = filter_input(INPUT_GET, 'ipv6', FILTER_SANITIZE_URL);
+
+function is_same_origin($ipv4, $ipv6) {
+    if (strpos($ipv4, ':') || strpos($ipv6, ':')){
+        return true;
+    }
+    return false;
+}
+
+if (is_same_origin($loc_ipv4, $loc_ipv6)) {
```

Comment:
Define single function argument . It is much more versatile than double or more argument.